src/marqo/s2_inference/clip_utils.py

```diff
@@ -29,6 +28,7 @@
 from marqo.s2_inference.processing.custom_clip_utils import HFTokenizer, download_model
 from marqo.s2_inference.types import *
 from marqo.tensor_search.enums import ModelProperties, InferenceParams
+from marqo.tensor_search.models.private_models import ModelAuth
```

Comment:
nit: This and the next line can be combined.

Reply:
done